warehouse/manage/views.py

```diff
@@ -1247,13 +1252,17 @@ def add_github_oidc_provider(self):
                 queue="success",
             )
 
+            self.metrics.increment("warehouse.oidc.add_github_provider.ok")
+
         return response
 
     @view_config(request_method="POST", request_param=DeleteProviderForm.__params__)
     def delete_oidc_provider(self):
         if not self.oidc_enabled:
             raise HTTPNotFound
 
+        self.metrics.increment("warehouse.oidc.delete_provider.attempt")
```

Comment:
```suggestion
        self.metrics.increment("warehouse.oidc.delete_provider.attempt", tags=["provider:github"])
```

and similar for others.

Reply:
`delete_provider` is generic over multiple provider types, so I'll figure out the best way to avoid hardcoding `github` here.

Reply:
c945ee7d